collect_app/src/main/java/org/odk/collect/android/utilities/ListViewUtils.java

```diff
@@ -57,4 +60,18 @@ public static void setAllToCheckedState(ListView lv, boolean check) {
             lv.setItemChecked(x, check);
         }
     }
+
+    // Function to toggle button label
+    public static void toggleButtonLabel(Button mToggleButton, ListView lv) {
+        if (ListViewUtils.selectedItemCount(lv) != lv.getCount())
+            mToggleButton.setText(R.string.select_all);
+        else
+            mToggleButton.setText(R.string.clear_all);
+    }
+
+    public static void disableToggleButton(Button mToggleButton, ListView lv) {
```

Comment:
See my comment on `DataManagerList`, I think it's better not to have a method for this. Otherwise it would better be named `disableToggleButtonIfListViewHasNoItems`!! 😊